config/routes.rb

```diff
@@ -26,6 +26,16 @@
     delete 'logga_ut' => 'devise/sessions#destroy', as: :destroy_user_session
   end
 
+  # tags and exlinks
+  resources :ex_links
+
+  namespace :admin do
+    get 'ex_links/del_unused_tags' => 'ex_links#del_unused_tags'
+    get 'ex_links/check_dead' => 'ex_links#check_dead'
+    get 'ex_links/check_expired' => 'ex_links#check_expired'
+    resources :ex_links
```

Comment:
They are same as before the change. I am getting error in all `link_to` parts of the code e.g.:
```
      <%= link_to ExLink.human_attribute_name(:delete_unused_tags),
                  {action: 'del_unused_tags'},
                  :data => { :confirm => 'Are you sure you want to delete unusued tags?' },
                  :class => 'btn danger' %>
```
that 
```
ActionController::UrlGenerationError at /admin/ex_links
No route matches {:action=>"del_unused_tags", :controller=>"admin/ex_links"}
```

Relevant parts of `rake routes` is I hope this one:

```
                          ex_links GET    /ex_links(.:format)                                      ex_links#index
                                   POST   /ex_links(.:format)                                      ex_links#create
                       new_ex_link GET    /ex_links/new(.:format)                                  ex_links#new
                      edit_ex_link GET    /ex_links/:id/edit(.:format)                             ex_links#edit
                           ex_link GET    /ex_links/:id(.:format)                                  ex_links#show
                                   PATCH  /ex_links/:id(.:format)                                  ex_links#update
                                   PUT    /ex_links/:id(.:format)                                  ex_links#update
                                   DELETE /ex_links/:id(.:format)                                  ex_links#destroy
     admin_ex_link_del_unused_tags GET    /admin/ex_links/:ex_link_id/del_unused_tags(.:format)    admin/ex_links#del_unused_tags
          admin_ex_link_check_dead GET    /admin/ex_links/:ex_link_id/check_dead(.:format)         admin/ex_links#check_dead
       admin_ex_link_check_expired GET    /admin/ex_links/:ex_link_id/check_expired(.:format)      admin/ex_links#check_expired
                    admin_ex_links GET    /admin/ex_links(.:format)                                admin/ex_links#index
                                   POST   /admin/ex_links(.:format)                                admin/ex_links#create
                 new_admin_ex_link GET    /admin/ex_links/new(.:format)                            admin/ex_links#new
                edit_admin_ex_link GET    /admin/ex_links/:id/edit(.:format)                       admin/ex_links#edit
                     admin_ex_link GET    /admin/ex_links/:id(.:format)                            admin/ex_links#show
                                   PATCH  /admin/ex_links/:id(.:format)                            admin/ex_links#update
                                   PUT    /admin/ex_links/:id(.:format)                            admin/ex_links#update
                                   DELETE /admin/ex_links/:id(.:format)                            admin/ex_links#destroy

```